android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/VPNService.kt

```diff
@@ -138,6 +138,15 @@ class VPNService : android.net.VpnService() {
         initializeGlean(Prefs.get(this).getBoolean("glean_enabled", false))
     }
 
+    override fun onCreate(savedInstanceState: Bundle) {
```

Comment:
This is not available for Services 😅
https://developer.android.com/reference/android/app/Service#onCreate()

Reply:
Yup, I learned :)